img/info.py

```diff
@@ -8,13 +8,16 @@
 filament_used_mm = 0
 time_s = 0
 layer_height = 0
+bottomLinesAnalyzedAmount = 400
```

Comment:
Can you make this naming snake case so it matches the other variable names?

Reply:
Sorry, .net habits ;)